examples/benchmarks/pytorch_bert_large.py

```diff
@@ -1,22 +1,35 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT license.
 
-"""Model benchmark example for bert-large.
+"""Model benchmark example for bert-large (24-layer, 1024-hidden, 16-heads, 340M parameters).
 
 Commands to run:
   python3 examples/benchmarks/pytorch_bert_large.py (Single GPU)
-  python3 -m torch.distributed.launch --nproc_per_node=8 examples/benchmarks/pytorch_bert_large.py (Distributed)
+  python3 -m torch.distributed.launch --use_env --nproc_per_node=8 examples/benchmarks/pytorch_bert_large.py \
+      --distributed (Distributed)
 """
 
-from superbench.benchmarks import Framework, BenchmarkRegistry
+import argparse
+
+from superbench.benchmarks import Platform, Framework, BenchmarkRegistry
 from superbench.common.utils import logger
 
 if __name__ == '__main__':
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        '--distributed', action='store_true', default=False, help='Whether to enable distributed training.'
+    )
+    args = parser.parse_args()
```

Comment:
expose all parameters to args? you could have `parameters=' '.join(sys.argv[1:])`
the example will be run as
```
python3 -m torch.distributed.launch --nproc_per_node=8 examples/benchmarks/pytorch_bert_large.py \
  --batch_size 1 --duration 120 --seq_len 128 --precision float32 --run_count 2
```

Reply:
I think it's just one example to show how to use, not for user to use directly, right?